README.md

```diff
@@ -13,12 +13,27 @@ Core repository for the platform team 4 is developing
 Running with Docker will ensure that users can quickly get environment up and running so they can quickly iterate on work. The application will work regardless of operating system and configuration.
 
 ## Web app
-Ensure you have set up backend env files first. Checkout backend README for more info.  
 
-1. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.  
+Ensure you have set up backend env files first. Checkout backend README for more info.
+
+1. Run `yarn run start:docker` to simultaneously run the frontend and backend in docker containers.
 
 ## Tests
 
-1. Run `yarn run test:docker` to simultaneously run the frontend and backend tests in docker containers. 
+1. Run `yarn run test:docker` to simultaneously run the frontend and backend tests in docker containers.
 
 For more detail, view `CONTRIBUTING.md`.
+
+# Generate LICENSE-3RD-PARTY.txt
```

Comment:
This should be a subtitle section? so ##

Reply:
This cannot be a subtitle unfortunately as this is not a part of the "Run with Docker" documentation.